tests/wallet/cat_wallet/test_cat_wallet.py

```diff
@@ -743,6 +743,105 @@ async def check_wallets(wallet_node):
         await time_out_assert(15, cat_wallet.get_confirmed_balance, 35)
         await time_out_assert(15, cat_wallet.get_unconfirmed_balance, 35)
 
+    @pytest.mark.parametrize(
+        "trusted",
+        [True, False],
+    )
+    @pytest.mark.asyncio
+    async def test_cat_coin_selection(self, two_wallet_nodes, trusted):
```

Comment:
I'm dubious that we need this test since we've changed the CAT select_coins algorithm to use the shared `select_coins` method.  Is there a reason why we should specifically test this for CATs?

Reply:
Nope, ill just remove it.